File: java/src/main/java/br/com/summa/zx/BitOutputStream.java
/**
 * Zeta-Xi Code - by Einar Saukas
 *
 * See: https://github.com/einar-saukas/Zeta-Xi-Code
 */

package br.com.summa.zx;

public interface BitOutputStream {
    // writes next bit (either zero or not zero)
    void writeBit(int bit);
}

File: kotlin/src/main/kotlin/br/com/summa/zx/BitOutputStream.kt
/**
 * Zeta-Xi Code - by Einar Saukas
 *
 * See: https://github.com/einar-saukas/Zeta-Xi-Code
 */

package br.com.summa.zx

interface BitOutputStream {
    // writes next bit (either zero or not zero)
    fun writeBit(bit: Int)
}